docassemble/juvenilesealing/data/questions/juvenile-sealing-intro-10.yml

```diff
@@ -35,18 +35,24 @@ code: |
       not_finishing_end
   else:
     progressor.increment()  # 1
-  
+
   progressor.increment()  # 2
-  
+
   needs_dockets = False
-  if ( not have_docket_copies_already is True ):
+  # If user doesn't answer yes for docket sheets, show user the docket sheet description
+  if ( not have_docket_copies_already ):
     progressor.increment()  # 3
     needs_dockets = True
-    if ( have_docket_copies_already is None ):  # They don't know
-      about_docket_sheets
-    if ( wants_dockets_first is True ):
-      progressor.increment()  # 4
-      not_finishing_end  # end
+    # If user wants to see docket sheet instructions
+    if ( wants_dockets_first ):
+      # User has not seen docket sheet instructions yet
+      if ( not request_docket_sheets ):
```

Comment:
Possibly me being confused about how docassemble works, but isn't `request_docket_sheets`  the same question as `certified_copies_page_text_question`?  It looks like `request_docket_sheets` only has a continue option, so I would expect that value to only ever be True or undefined (prompting docassemble to then show the question), so think only the else clause would ever be invoked

Reply:
I imagine we could do something like `if (wants_dockets_first and force_ask(request_docket_sheets)):` on line 47 instead

Reply:
`force_ask` (or whatever the function to force a question to be asked even if it has already been answered) might help the code flow issue you found.
Putting `request_docket_sheets` inside the if block instead of the conditional might be clearer, but I personally like the idea of questions all resolving to values we actually use

Reply:
`force_ask` doesn't fix the code flow issue, I just tried it. All other suggestions appreciated.

Reply:
Hmm. I can do more experimentation. I do know another method that's less concise - you can manually undefine a value. That will definitely force it to be asked again.

Reply:
Also, if `force_ask` is not working, it may be worth going onto the Slack channel and asking what's up with that.